app/javascript/packages/verify-flow/steps/password-confirm/forgot-password.tsx

```diff
@@ -0,0 +1,52 @@
+import { useContext } from 'react';
+import { PageHeading, Button } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
+import { getAssetPath } from '@18f/identity-assets';
+import { FormStepsContext } from '@18f/identity-form-steps';
+
+interface ForgotPasswordProps {
+  goBack: () => void;
+}
+
+export function ForgotPassword({ goBack }: ForgotPasswordProps) {
+  const { resetPasswordUrl } = useContext(FormStepsContext);
+
+  function goToResetPassword() {
+    window.location.href = resetPasswordUrl!;
+  }
+
+  return (
+    <>
+      <img
+        src={getAssetPath('status/info-question.svg')}
+        width="50"
+        alt=""
+        height="50"
+        className="margin-bottom-4"
+      />
+      <PageHeading>{t('idv.forgot_password.modal_header')}</PageHeading>
+      <ul className="usa-list">
+        {t('idv.forgot_password.warnings').map((warning) => (
+          <li key={warning}>{warning}</li>
+        ))}
+      </ul>
+      <p>
+        <Button
+          className="margin-top-4"
+          isBig
+          isWide
+          onClick={() => {
+            goBack();
+          }}
+        >
+          {t('idv.forgot_password.try_again')}
+        </Button>
+      </p>
+      <p>
+        <Button isBig isOutline isWide onClick={() => goToResetPassword()}>
+          {t('idv.forgot_password.reset_password')}
+        </Button>
+      </p>
```

Comment:
Since this looks mostly the same as the [existing StatusPageComponent Rails implementation](https://github.com/18F/identity-idp/blob/main/app/components/status_page_component.html.erb), maybe we should align the markup and/or create a corresponding React component. Also, I don't necessarily know or trust that default paragraph margins are identical to the precise margins we expect on these page layouts (2rem above button, 1rem between buttons).

```suggestion
      <div className="margin-top-4">
        <Button
          isBig
          isWide
          onClick={goBack}
        >
          {t('idv.forgot_password.try_again')}
        </Button>
      </div>
      <div className="margin-top-2">
        <Button isBig isOutline isWide onClick={goToResetPassword}>
          {t('idv.forgot_password.reset_password')}
        </Button>
      </div>
```